.github/workflows/deploy.yml

```diff
@@ -31,7 +34,12 @@ jobs:
           node-version: 16
 
       - name: Install of node dependencies
-        run: npm ci
+        run: npm ci --legacy-peer-deps
```

Comment:
FYI
I suppose we need to cut this in the future, because it breaks some basic commands like `npm audit`, etc.
As I see it is connected to `react-image-video-lightbox`, so may be we need to replace this dependency.